Parse/Internal/HttpClient.Unity.cs

```diff
@@ -140,7 +140,11 @@ internal class HttpClient : IHttpClient {
         return (HttpStatusCode)201;
       }
       String errorCode = Regex.Match(www.error, @"\d+").Value;
-      return (HttpStatusCode)Int32.Parse(errorCode);
+      int errorNumber = 0;
+      if (!Int32.TryParse(errorCode, out errorNumber)) {
+        return (HttpStatusCode)201;
```

Comment:
Hmm. At this point `www` will return an error and we should return one, otherwise we'll break `ParseCommandRunner` https://github.com/ParsePlatform/Parse-SDK-dotNET/blob/master/Parse/Internal/Command/ParseCommandRunner.cs#L43. `201` isn't really an error.

I always wonder why they can't parse the string. Maybe the string doesn't contain number at all, which means it's a client error. By default we should return `400` https://github.com/hungrylabs/hungry/blob/gogo.push-xamarin-ios/windows/Parse/Parse/PlatformHooks.Unity.cs#L678